man/systemd.network.xml

```diff
@@ -706,6 +706,12 @@
             <citerefentry project='man-pages'><refentrytitle>inet_pton</refentrytitle><manvolnum>3</manvolnum></citerefentry>.</para>
           </listitem>
         </varlistentry>
+        <varlistentry>
+          <term><varname>OnDevice=</varname></term>
+          <listitem>
+            <para>A boolean. Specifies whether to take device as default route.</para>
```

Comment:
we shoul really grok this first... Maybe use wireshark to see what happens depending on the route?